doc/instruction_decode_execute.rst

```diff
@@ -165,3 +162,13 @@ Source File: :file:`rtl/ibex_load_store_unit.sv`
 
 The Load-Store Unit (LSU) interfaces with main memory to perform load and store operations.
 See :ref:`load-store-unit` for more details.
+
+.. rubric:: Footnotes
+
+.. [#B_draft] Ibex fully implements draft version 0.92 of the RISC-V Bit Manipulation Extension.
+   This extension may change before being accepted as a standard by the RISC-V Foundation.
+   Ultimately, Ibex will be updated accordingly to implement the final standard extension.
+   However, designs based on the current version of Ibex may not get support for bit manipulation instructions from future software toolchains.
+
+.. [#B_zb_tmp] The sign-extend instructions `sext.b/sext.h` are defined but not yet categorized in draft version 0.92 of the extension.
```

Comment:
I suppose this is a fix for another PR, but I'm seeing these under Zbb in my version of the 0.92 spec?

Reply:
They are not unambiguously categorized: The table of contents lists them as part of Zbb, whereas the famous figure on Page 3 does not show them at all. Most likely this is just an oversight. The changelog shows that these instructions were added on the very last day.
We discussed this with RISCV-DV because we wanted the categorization inside Ibex to be aligned with the verification framework. Otherwise you could have a Zbb test that never generates a `sext.b/h` instructions because RISCV-DV categorizes them differently.
The current working draft categorizes them in Zbb consistently. But in addition, also some other instructions got assigned new categories. In the end, we didn't want to make any assumptions and decided together to stick to 0.92, to keep `sext.b/h` in `Zb_tmp` for now, and document this accordingly.

Reply:
I clarified in the documentation, that this categorization also matches the verification framework.